### ISSUE TITLE
[SIEM] Remove deprecated code from 8.0

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#19039]
I looked through the Security Ingest modules/beats for comments and TODOs indicating that a change was needed for 8.0. This may not be an exhaustive list so if you know of change that should be made for 8.0 please add it hear.

**Winlogbeat**
- [ ] Remove hash.* fields
  https://github.com/elastic/beats/blob/d11d6092d0dcd505c8bba0e7a12ceb6d7b02700b/x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js#L428-L429

- [ ] Remove the eventlogging API reader implementation.
  https://github.com/elastic/beats/blob/d81ef7368b0ef321ff33b42be877c4aa1f1c09e0/winlogbeat/eventlog/eventlogging.go#L281

**Auditbeat**
 - [ ] Remove auditd module event categories
    https://github.com/elastic/beats/blob/bd7414d09b9589f9843193a53f8bfef9f9eca0a4/auditbeat/module/auditd/audit_linux.go#L579

- Remove FIM `hash.*` fields
    https://github.com/elastic/beats/blob/56ba9d089cb9441af65caf4cd534f1aec1726cbc/auditbeat/module/file_integrity/event.go#L308-L309
    https://github.com/elastic/beats/blob/56ba9d089cb9441af65caf4cd534f1aec1726cbc/auditbeat/module/file_integrity/event_test.go#L318-L320

**Comment by peasead** [comment]
An additional gap for Packetbeat is that `event.type` is blank. 

My thought is that `event.type` should reflect what `type` currently is and that `event.action` would carry the `start` and `end` values?

**Comment by jamiehynds** [comment]
@andrewkroh is this issue still relevant for 8.0?

**Comment by andrewkroh** [comment]
Yes, this is relevant. I went through and checked the boxes for the ones that have been address, but the others still should be changed for 8.0.

**Comment by efd6** [comment]
@andrewkroh Is [this coredns deprecation](https://github.com/elastic/beats/blob/6e69b0578bc654dae0f09fc71fd9d5d639a7914a/x-pack/filebeat/module/coredns/log/ingest/pipeline-entry.yml#L75-L102) relevant here? If so, what else is required?

**Comment by andrewkroh** [comment]
@efd6 Yes, that looks relevant. I've added a bullet point to the description for this. I think what needs done is to remove those processors from the pipeline, remove the related fields from the _meta/fields.yml file, and then run `make update` to regenerate some files and docs.

**Comment by efd6** [comment]
@andrewkroh It mentions removing `coredns.dnssec_ok`, so that should be hosed out too?

**Comment by andrewkroh** [comment]
Yes, `coredns.dnssec_ok` can be removed because we have `dns.header_flags` in the event (an ECS field) and users can check for the `DO` flag that indicates "DNSSEC answer OK".
